src/webview/css/css.js

```diff
@@ -157,20 +159,22 @@ hr {
   flex-direction: row;
 }
 .stream-text,
-.topic-text,
+.topic-content,
 .private-header {
   padding: 0 0.5em;
   line-height: 2;
   white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
 }
-.topic-text {
+.topic-content {
   flex: 1;
   padding-left: 0.5em;
   background: #ccc;
   min-width: 30%;
 }
+.topic-text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
```

Comment:
Adding some transparency to the text (about 0.5) will make it look better, as this is secondary information to the topic name.

Reply:
Agreed, done!